src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -1301,6 +1301,15 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
                     }
                 };
 
+                clrPrivate.LoaderHeapAllocRequest += delegate(LoaderHeapAllocRequestTraceData data)
+                {
+                    var process = data.Process();
+                    var stats = currentManagedProcess(data);
+                    var _method = JITStats.JitAllocComplete(stats, data);
+                    //TODO: What to do here of _method??
```

Comment:
You should not need to do anything with `_method` here.  `JitAllocComplete` did all the work to capture the allocation.